pkgs/applications/misc/synergy/default.nix

```diff
@@ -1,82 +1,37 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, fetchurl, cmake, xlibsWrapper
-, ApplicationServices, Carbon, Cocoa, CoreServices, ScreenSaver
-, libX11, libXi, libXtst, libXrandr, xinput, curl, openssl, unzip }:
+, ApplicationServices, Carbon, Cocoa, CoreServices, ScreenSaver, avahi
+, libX11, libXi, libXtst, libXrandr, xinput, curl, openssl, unzip, qtbase }:
 
 stdenv.mkDerivation rec {
   pname = "synergy";
-  version = "1.8.8";
+  version = "1.11.0";
 
+  # FIXME: tags for versions usually begin with a v, however 1.11.0 is
+  # missing this v, so temporarily remove it, however if they readd it
+  # make sure to readd it here accordingly
```

Comment:
I think this comment is unnecessary and will be noise for people searching for FIXME, since there’s nothing we can fix. For all we know there will be no more v tags going forward, and if it changes it’s trivial to find the new tag and update. It happens all over the place.